pkg/supportbundle/clusterspec/embed.go

```diff
@@ -0,0 +1,39 @@
+package clusterspec
+
+import (
+	_ "embed"
+
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"github.com/replicatedhq/troubleshoot/pkg/supportbundle"
+)
+
+//go:embed spec.yaml
+var raw []byte
+
+var spec *troubleshootv1beta2.SupportBundle
+
+func init() {
+	var err error
+	spec, err = supportbundle.ParseSupportBundleFromDoc(raw)
+	if err != nil {
+		panic(err)
```

Comment:
I think we need a better way to handle these errors. KOTS should not crash if there is a problem loading specs. I suggest we have the specs lazily loaded in the `Getters` and return an error if there is a failure. This is how I would do this

```go
var (
	defaultSpec *troubleshootv1beta2.SupportBundle
)

func GetDefault() (*troubleshootv1beta2.SupportBundle, error) {
	// Some worth improvements
	// * Thread safety
	// * sync.Pool for spec objects. Maybe? Perhaps making object copies is intentional.
	if (spec == nil) {
		var err error
		defaultSpec, err = supportbundle.ParseSupportBundleFromDoc(raw)
		if err != nil {
			return nil, err
		}
	}
	return defaultSpec.DeepCopy(), nil
}
```

Reply:
This change would require additional application error handling like showing some information in Admin console's troubleshoot page. There is an [error message response object](https://github.com/replicatedhq/kots/blob/main/web/src/components/troubleshoot/GenerateSupportBundleModal.tsx#L210) used by the web UI to show that there was a problem. Perhaps this will just work but its worth testing